src/utils/helper.ts

```diff
@@ -112,6 +113,26 @@ export function ToJsonString(_in: any): string {
     return JSON.stringify(_in).split(/[\r\n]+/).join("").split("\\").join("\\\\").split("'").join("\\'")
 }
 
+export function ToPythonString(value: any, type: string): string {
+    let str = value;
+
+    if (!isNullOrUndefined(str)) {
+        if (type == 'boolean') {
+            if (Boolean(value)) {
+                str = "True";
+            }
+            else {
+                str = "False";
+            }
+        }
+        else {
+            str = JSON.stringify(str)
```

Comment:
Can this handle number type or normal string type?

Reply:
Yes